usr/src/uts/common/fs/zfs/sys/dmu.h

```diff
@@ -564,12 +564,7 @@ typedef struct dmu_buf_user {
  * NOTE: This function should only be called once on a given dmu_buf_user_t.
  *       To allow enforcement of this, dbu must already be zeroed on entry.
  */
-#ifdef __lint
-/* Very ugly, but it beats issuing suppression directives in many Makefiles. */
```

Comment:
Is this really "better"?  You're trading away the ability to notice unused statics, which seems to me a high price to pay to get rid of an #ifdef __lint.